regression-test/data/nereids_p0/aggregate/agg_nullable_2.out

```diff
@@ -476,6 +476,15 @@ false
 -- !select_array_agg_n --
 ["string"]
 
+-- !select_regr_r2 --
+0
+
```

Comment:
Hello @zclllyybb , I'm not entirely sure, but I think after `be_exec_version >= 5` (which is now), this should be `0` instead of `\N`, right?

```sql
select regr_r2(kint, kint) from agg_nullable_test_2;
```

There's only one row of data in the table, and in PostgreSQL, this would return `null`.